test/web-platform/views/index.ejs

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.1.4/semantic.css" rel="stylesheet"
+          type="text/css"/>
+    <script src="http://code.jquery.com/jquery-1.10.2.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.1.4/semantic.js" type="text/javascript"></script>
+    <style type="text/css">
+        body > .grid {
+            height: 100%;
+        }
+
+        body > .grid > .column {
+            width: 800px !important;
+        }
+    </style>
+</head>
+<body>
+<div class="ui middle aligned center aligned grid">
+    <div class="column">
+        <div class="ui segments">
+            <div class="ui basic center aligned segment">
+                <h3 class="ui header">Web platform tests</h3>
+            </div>
+            <div class="ui basic left aligned segment">
+                <ul class="ui list">
+                    <% for (var i = 0; i < tests.length; i++) { %>
```

Comment:
Can we use Mustache instead of EJS? We've switched to Mustache everywhere. For now we only have EJS templates in playground (just a couple of lines) and here. Can we switch here and in playground and remove extra dependency?